package.json

```
@@ -3,28 +3,18 @@
   "author": "Nomic Foundation",
   "license": "SEE LICENSE IN EACH PACKAGE'S LICENSE FILE",
   "private": true,
-  "workspaces": {
-    "packages": [
-      "packages/core",
-      "packages/ui",
-      "packages/hardhat-plugin",
-      "packages/hardhat-plugin-ethers",
-      "packages/hardhat-plugin-viem",
-      "examples/*"
-    ]
-  },
   "scripts": {
-    "build": "npm run build -w packages",
-    "lint": "npm run lint --workspaces",
+    "build": "pnpm --filter \"./packages/*\" build",
+    "lint": "pnpm --filter \"./packages/*\" lint",
     "postlint": "prettier --check *.{md,json} ./config/**/*.{js,json}",
-    "lint:fix": "npm run lint:fix --workspaces",
+    "lint:fix": "pnpm --filter \"./packages/*\" lint:fix",
     "postlint:fix": "prettier --write *.{md,json} ./config/**/*.{js,json}",
-    "test": "npm run test -w packages && npm run test:integrations -w packages/core",
-    "test:coverage": "npm run test:coverage -w packages",
-    "test:examples": "npm run test -w examples --if-present",
+    "test": "pnpm --filter \"./packages/*\" test && pnpm --filter \"./packages/core\" test:integrations",
+    "test:coverage": "pnpm --filter \"./packages/*\" test:coverage",
+    "test:examples": "pnpm --filter \"./examples/*\" test",
     "watch": "tsc --build --watch packages/core packages/hardhat-plugin",
-    "fullcheck": "npm run build && npm run lint && npm run test:examples",
-    "clean": "npm run clean --workspaces --if-present"
+    "fullcheck": "pnpm build && pnpm lint",
```

```suggestion
    "fullcheck": "pnpm build && pnpm lint && pnpm test",
```
Even if we are leaving out the example tests we should run the standard integeration tests.